doc/guides/maintaining-npm.md

```diff
@@ -1,5 +1,11 @@
 # Maintaining npm in Node.js
 
+New pull requests should be opened when a "next" release has been released.
+
+Once the "next" release has been promoted to "latest" the PR should be updated as neccessary
```

Comment:
Nits:
neccessary -> necessary
Missing period.
Wrapping this and the next line at 80 chars.